### ISSUE TITLE
Update pysftp

### ISSUE THREAD

**Issue by mhils** [mitmproxy/mitmproxy#2737]
This requires that I update the snapshot server to have a consistent hostkey: This is now documented at https://hub.docker.com/r/atmoz/sftp/.

**Comment by mhils** [comment]
The server should now have a static hostkey:
```
root@docker ~/d/snapshots.mitmproxy.org# cat ssh_host_ed25519_key.pub
ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAwWaZQamQPW4Ve5jWhvDSBAgS1ptPNUi7wPvOAKYFl7 root@docker
root@docker ~/d/snapshots.mitmproxy.org# cat ssh_host_rsa_key.pub
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDM+76ILWIXTdzR6cSwhWe70de9Wca9+uMskJ1Ol9YQrVOX1LvBJSGYKLZwOtC/8Q+T+JvvuNZYJJz1cNFruSRuHH+87UKJ/KDoKZ5Q6X9LBeZpnsclvzDnQI4sJwK+434D38YGUJsnHhZ3+BdPGkGmrxj/i/McCPbjXBDlnjYYoaaigcvF7conVhPG8O825tXFGc3wpqEP6LVF6HcWMgF04mY4SKvM6x95wmtGZKNraU2eZWqIrpS6mjdsuCk/csem9A+nP8BQrCpV1B5Bc0TbsaeUxReN1slajwgXahkTs5bvqMP2nHSEE7Z5XxNlIzhzmrOQUtibG4b7pjFMH/XePhnhOyCmPk6RGc4c5RBwwOyrIbbi15wv372150E2pqW1vf9xZRnoLbe8fC4yqZw5k3gimDAJ2AXr/OZ7MzdeBaDqtBBU+wliLbGjIEuqXIPMeljbd2XU5ZcPA4aBvWhURYEHcf3EEAn+TX0yi6qat0LXGTe8NkJQPseTyGddp9ULJPr1A6iFOBSrw0LTiY3NAI5DseYP+Abj4wvgrwymBlM00VDDyLH5F1DpQ0R1OIwDfvdcqntfdxk8iraZCnCs3AyYeE/PXYt/BgqkMVZBcGp8h7r4jxNsDk8NiUAY73FRHYz7fmvZ2dyrxo4gWSLSiLptHGRISJEySfTms6ppDw== root@docker
```

**Comment by mhils** [comment]
Fixed by 09f6160645d3854f960826d5c17992dedb6fa48a
